skops/io/_sklearn.py

```diff
@@ -97,15 +99,23 @@ def BaseEstimator_get_instance(state, src):
     return instance
 
 
+def bunch_get_instance(state, src):
+    # Bunch is just a wrapper for dict
+    content = dict_get_instance(state, src)
+    return Bunch(**content)
+
+
 # tuples of type and function that gets the state of that type
 GET_STATE_DISPATCH_FUNCTIONS = [
     (Tree, BaseEstimator_get_state),
+    (Bunch, dict_get_state),
```

Comment:
do we need this? Doesn't the dict only handle this?

Reply:
Yes, right